src/server/schema.ts

```diff
@@ -63,6 +66,20 @@ export const schema = `
       timeoutMs: Int = 10000
     ): [LookupQueryResult]
   }
+  
+  """
+  The mode in which the literal search query (\`query\`) is interpreted before it is sent to the term sources.
+  """
+  enum QueryMode {
+    "This mode is only for backwards-compatibility. Switch to \`SMART\` mode for forwards-compatbility."
+    DEPRECATED @deprecated(reason: "This mode will be removed in the future. Use \`SMART\` instead.")
+    
+    "Connect search query words with a boolean AND operator and optimise query input for term sources."
```

Comment:
Or we could generalize this a bit by merely saying that this query mode takes care of query optimization, without mentioning its details (e.g. that it uses `AND` - that could change if we, at some point, decide to query Virtuoso differently)?